testsuite/tests/mgc/test_basic.py

```diff
@@ -0,0 +1,105 @@
+"""
+This module contains the very basic tests and their dependencies for MGC
+"""
+from time import sleep
+import httpx
+import pytest
+from testsuite.openshift.httpbin import Httpbin
+from testsuite.openshift.objects.gateway_api import GatewayLight, HTTPRoute
+
+
+# prerequisites:
+# this module assumes the hub cluster is also a spoke cluster so that everything happens on the only cluster
+#
+# multi-cluster-gateways ns is created and set as openshift["project"]
+# managedclustersetbinding is created in openshift["project"]
+# placement named "local-cluster" is created in openshift["project"] and bound to clusterset
+# gateway class "kuadrant-multi-cluster-gateway-instance-per-cluster" is created
+# openshift2["project"] is set
+#
+# dnspolicies are created and bound to gateways automatically by mgc operator
+# dnspolicies leak at this moment
+
+
+@pytest.fixture(scope="module")
+def mgc_apex_domain(openshift):
```

Comment:
what is apex?

Reply:
> what is apex?
free to play battle royale? :-)

Reply:
An apex domain is a root domain that does not contain a subdomain part. Something we would call a base domain I believe. If required we can change the naming to base domain. In this context it is used to describe a domain from which mgc creates subdomains.

Reply:
Yeah, I would definitely change it to `base domain` or